### PR TITLE
Created production apply related infra

### PR DESCRIPTION
**Pull request by mrzta**
## Context

Create the Production environment in the AKS cluster

## Changes proposed in this pull request

- Create production global_config file
- Add production_aks to Makefile
- Add production_aks tfvar files
- Add to deploy_v2 workflow

## Other changes outside this pull request
- Add APPLY-APP-SECRETS-PRODUCTION secret from s121 to s189 key vault modifying values to target sandbox2
- Add BAT-INFRA-SECRETS-PRODUCTION secret from s121 to s189 key vault clearing the CF password and user values
- Add APPLY_CYPRESS-SECRETS secret from s121 to s189 key vault

## Guidance to review

- Run command make production_aks deploy CONFIRM_PRODUCTION=1 DISABLE_PASSCODE=1
- AKS Sandbox environment accessible onhttps://www2.apply-for-teacher-training.service.gov.uk

## Link to Trello card

[](https://trello.com/c/6aUrphbe/97-apply-include-aks-in-apply-cd-pipeline-production-sandbox)

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Comment by mrzta** [comment]
[Apply - Include AKS in Apply CD Pipeline - Production/Sandbox](https://trello.com/c/6aUrphbe/97-apply-include-aks-in-apply-cd-pipeline-production-sandbox)

File: terraform/modules/kubernetes/main.tf [hunk 1]
```diff
@@ -18,7 +18,7 @@ resource "kubernetes_deployment" "webapp" {
       }
       spec {
         node_selector = {
-          "kubernetes.azure.com/agentpool" = "applications"
+          "kubernetes.azure.com/agentpool" = "apps2"
```
Comment (saliceti) [inline terraform/modules/kubernetes/main.tf:21]: Should we use the new tag here? CC @smneal

Comment (smneal) [reply terraform/modules/kubernetes/main.tf:21]: The new tag is `"teacherservices.cloud/node_pool": "applications"` but it's only in the production cluster at the moment.

Comment (smneal) [reply terraform/modules/kubernetes/main.tf:21]: https://github.com/DFE-Digital/teacher-services-cloud/pull/47

**Comment by smneal** [comment]
Should we add continue on error to the production one as well? @saliceti @RMcVelia as per https://github.com/DFE-Digital/apply-for-teacher-training/blob/e3a2c8eeaa21d95a4c449f23650cd108c1782d84/.github/workflows/build-and-deploy.yml#LL656C10-L656C10 ?

**Comment by RMcVelia** [comment]
i would think so (continue-on-error: true) as we don't want to fail the workflow, and deploy-aks-after-production could now need deploy-aks-production instead of deploy-production

**Review by smneal — APPROVED** [review]
LGTM

**Pull request merged**